tests/integration_tests/tests/utils.py

```diff
@@ -167,13 +168,43 @@ def wait_for_deployment_deletion_to_complete(
                client=client)
 
 
+def wait_for_blueprint_to_upload(
+        blueprint_id, client, timeout_seconds=30):
+    do_retries(func=verify_blueprint_uploaded,
+               timeout_seconds=timeout_seconds,
+               blueprint_id=blueprint_id,
+               client=client)
+
+
+def wait_for_blueprint_upload_to_end(
```

Comment:
Don't we need jsut one of these functions?

Reply:
One passes for any end_state, the other only for uploaded. I can parametrize them if you want

Reply:
done